gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_2_1/schema.yml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.2.1
+      author: GraviteeSource Team
+      changes:
+        # ################
+        # OrganizationLicense changes
+        # ################
+        - createTable:
```

Comment:
Do you think adding an index on `organizationId` could be valuable?

Reply:
No longer necessary because the organization id will be used as the id. Should be indexed automatically by JDBC + Mongo 👍